RandomStudio/random-studio
# Show placeholder image when video loading

- When video is loading, show a static placeholder image
- Minimise page layout jumping around once video loads

**Steps to test**
1.  Reload (without cache) the homepage, before this fix you will see the header video has 0px height, then jumps to full height
2. With this fix, it should start at full height and show a static placeholder image while the video loads

**Note**
The current home video reports its height via BunnyCDN slightly incorrect. You may see a very slight jump still, but this should be only 50px~. You can also check the Rampvlucht project thumbnail video on the projects page.
